shared/util_linux.go

```diff
@@ -13,6 +13,7 @@ import (
 	"path/filepath"
 	"reflect"
 	"strings"
+	"syscall"
```

Comment:
> NOTE: Most of the functions, types, and constants defined in this package are also available in the [golang.org/x/sys](https://pkg.go.dev/golang.org/x/sys) package. That package has more system call support than this one, and most new code should prefer that package where possible. See https://golang.org/s/go1.4-syscall for more information.

Should we use [golang.org/x/sys](https://pkg.go.dev/golang.org/x/sys) instead?

Reply:
Casting will only work if using `syscall.WaitStatus`

Reply:
Oh i missed this comment, ok ta